openslides/core/urls.py

```diff
@@ -16,9 +16,9 @@
         views.VersionView.as_view(),
         name='core_version'),
 
-    url(r'^angular_js/(?P<openslides_app>site|projector)/$',
-        views.AppsJsView.as_view(),
-        name='core_apps_js'),
+    url(r'^openslides_javascript/(?P<realm>site|projector)/$',
```

Comment:
This URL does not only provide AngularJS related stuff but also our third party libs and the like.